library/deletedrug.php

```diff
@@ -27,11 +28,51 @@
     if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
         CsrfUtils::csrfNotVerified();
     }
+
+    /**
+     * find the drug name in the prescription table
+     */
+    try {
+        $drug_name = "SELECT patient_id, drug FROM prescriptions WHERE id = ?";
+        $dn = sqlQuery($drug_name, [$id]);
+    } catch (Exception $e) {
+        echo 'Caught exception ', text($e->getMessage()), "\n";
+        if ($e->getMessage()) {
+            exit;
+        }
+    }
+
+    /**
+     * remove drug from the medication list if exist
+     */
+    try {
+        $pid = $dn['patient_id'];
+        $drugname = $dn['drug'];
+        if (!empty($drugname)) {
+            $medicationlist = "DELETE FROM lists WHERE pid = ? AND type = 'medication' AND title = ?";
+            sqlStatement($medicationlist, [$pid, $drugname]);
+            $table = 'prescriptions';
+            $logstring = 'Medication ' . $drugname . 'removed from chart' . $pid;
+            EventAuditLogger::instance()->newEvent("delete", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "$table: $logstring");
```

Comment:
include the $pid in the 6th parameter,
```
EventAuditLogger::instance()->newEvent("delete", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "$table: $logstring", $pid);
```
(and then really don't need it in the $logstring)

Reply:
@bradymiller 
with all due respect, the PID is in the log string. It would appear that the purpose of the log string is to add clarity to the entry. Without the log string, there would be no entry as to which drug was deleted from the patient's chart. 
So, is a change that would replace the log string and table can be dropped also.

     `EventAuditLogger::instance()->newEvent("delete", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "$drugname", $pid);`

Reply:
hi @juggernautsei , should get a bit more context on the log entry (ie. that it is a drug and not an allergy; i'll bring this in and add something to it). thanks for the improvement! -brady